http-client/src/main/java/io/micronaut/http/client/netty/NettyClientHttpRequest.java

```diff
@@ -282,7 +282,8 @@ public FullHttpRequest toFullHttpRequest() {
                 req = new DefaultFullHttpRequest(
                         HttpVersion.HTTP_1_1,
                         method,
-                        uriStr
+                        uriStr,
+                        false
```

Comment:
Why is header validation disabled here?

Related: https://github.com/micronaut-projects/micronaut-core/security/advisories/GHSA-694p-xrhg-x3wm

Reply:
The headers were already validated in `headers.getNettyHeaders()` so we know it is safe